app/components/plaid-link.js

```diff
@@ -0,0 +1,12 @@
+import PlaidLink from 'ember-plaid/components/plaid-link';
+import config from '../config/environment';
+
+const plaidConfig = config['ember-plaid'];
+
+export default PlaidLink.extend({
+  clientName: plaidConfig.clientName,
+  product: plaidConfig.product,
+  key: plaidConfig.key,
+  env: plaidConfig.env,
+  selectAccount: plaidConfig.selectAccount
```

Comment:
we also want to support modifying all of the other variables above such as webhook etc.

Reply:
not sure how this component object is used though compared to above.